querybook/webapp/redux/board/action.ts

```diff
@@ -213,3 +289,60 @@ export function deleteBoardItem(
         });
     };
 }
+
+export function updateBoardItemDescription(
```

Comment:
updateBoardItemDescription and updateBoardItemMeta can be combined

`updateBoardItem(boardItemId: number, fields: { description?: ..., meta?: ... })`